pkg/migration/runner_test.go

```diff
@@ -240,20 +240,19 @@ func TestOnline(t *testing.T) {
 	)`
 	testutils.RunSQL(t, table)
 	m, err = NewRunner(&Migration{
-		Host:         cfg.Addr,
-		Username:     cfg.User,
-		Password:     cfg.Passwd,
-		Database:     cfg.DBName,
-		Threads:      16,
-		Table:        "testonline3",
-		Alter:        "ADD INDEX(b)",
-		ForceInplace: true,
+		Host:     cfg.Addr,
+		Username: cfg.User,
+		Password: cfg.Passwd,
+		Database: cfg.DBName,
+		Threads:  16,
+		Table:    "testonline3",
+		Alter:    "ADD INDEX(b)",
 	})
 	assert.NoError(t, err)
 	err = m.Run(t.Context())
 	assert.NoError(t, err)
 	assert.False(t, m.usedInstantDDL) // not possible
-	assert.True(t, m.usedInplaceDDL)  // as
+	assert.False(t, m.usedInplaceDDL) // ADD INDEX operations now use copy process for replica safety
```

Comment:
```suggestion
	assert.False(t, m.usedInplaceDDL) // ADD INDEX operations now always require copy
```

The replica safety thing is true (when non Aurora), but neither here nor there. The behavior here is also strongly correlated to `ForceInplace`.